bitnami/argo-cd/templates/notifications/bots/slack/rolebinding.yaml

```diff
@@ -0,0 +1,18 @@
+{{ if and .Values.notifications.enabled .Values.notifications.bots.slack.enabled .Values.rbac.create }}
```

Comment:
If you add the same check to different files like for example _bitnami/argo-cd/templates/notifications/bots/slack/role.yaml_, it would be great to follow the same check order to follow a structure. For example, always use this order:  _.Values.rbac.create .Values.notifications.enabled .Values.notifications.bots.slack.enabled_